### ISSUE TITLE
Allow user to change search engine

### ISSUE THREAD

**Issue by sskyy** [beakerbrowser/beaker#1033]
Operation System: MacOS
Beaker version: 0.8.0-prerelease.6

I could not find a place to change search engine, and I find that duckduckGO is hard coded in the program. I would be happy to give a PR if you approve.

**Comment by pfrazee** [comment]
Yeah this has been on my todo list. I'd be happy to hit it myself, or to accept a PR.
